deptrac.yaml

```diff
@@ -116,19 +116,19 @@ deptrac:
       - File
```

Comment:
Why is this file still needed?

Reply:
Do mean the deptrac.yaml file or the 'File' config?